zammad/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: zammad
-version: 6.4.1
+version: 6.4.2
```

new feature should raise minor version.
```suggestion
version: 6.5.0
```